src/foremast/utils/subnets.py

```diff
@@ -27,7 +27,7 @@
 
 
 # TODO: split up into get_az, and get_subnet_id
-@retries(max_attempts=6, wait=2.0, exceptions=SpinnakerTimeout)  # noqa
+@retries(max_attempts=10, wait=6.0, exceptions=SpinnakerTimeout)  # noqa
```

Comment:
Same comment as security_group.py

Reply:
I have replaced this with the backoff library per my initial recommendation in my pull request. I have changed the max attempts back to the original values per request